src/background/background.ts

```diff
@@ -44,7 +44,14 @@ chrome.runtime.onConnect.addListener(port => {
           const path = ScreenPaths.GENERATION;
           const urlParams = new URLSearchParams();
           urlParams.set('authRequest', payload);
-          popupCenter({ url: `/popup.html#${path}?${urlParams.toString()}` });
+          // For integration tests, launches in full-page instead of popup
+          if (IS_TEST_ENV) {
+            await chrome.tabs.create({
+              url: chrome.runtime.getURL(`full-page.html#${path}?${urlParams.toString()}`),
+            });
```

Comment:
I'd suggest a descriptive method here, rather this duplicated comment explain what's going on. 

```ts
if (IS_TEST_ENV) {
  await openAuthRequestInFullPage()
}
```

With the code above everything that's explained in the comment is clear in the code, no comment necessary. 

Perhaps one comment worth noting—that could go inside the body of the helper method—is the _why_, rather than the _what_. Which is, because playwright doesn't support popups properly? Is there a Github issue you could link to in the comment?

Where possible I think we should also avoid using any `chrome.*` api directly, too, which a helper method would also address.